docs/u2f_migration.md

```diff
@@ -0,0 +1,157 @@
+# Migrating from U2F to WebAuthn
+
+The Chromium team [recommends](https://groups.google.com/a/chromium.org/forum/#!msg/security-dev/BGWA1d7a6rI/W2avestmBAAJ)
+application developers to switch from the U2F API to the WebAuthn API. 
+This document describes how such a migration for Ruby application using the [u2f gem by Castle](https://github.com/castle/ruby-u2f) may be achieved.
+
+Note that the migration is one way: credentials registered using WebAuthn cannot be made compatible with the U2F API.
```

Comment:
```suggestion
Note that the migration is one-way: credentials registered using WebAuthn cannot be made compatible with the U2F API.
```

(To me, "one way" looks like it means "one of multiple ways" rather than "non-reversible".)